cmd/src/server.rs

```diff
@@ -632,6 +635,7 @@ impl<ER: RaftEngine> TiKVServer<ER> {
             self.system.take().unwrap(),
             &server_config,
             raft_store,
+            Arc::new(VersionTrack::new(self.config.coprocessor.clone())),
```

Comment:
I think it needs to be the same as Line 611, also `Arc<RwLock<Config>>` is better than `Arc<VersionTrack<Config>>` if we don't the `Tracker`